obi_one/endpoints/declared_endpoints.py

```diff
@@ -0,0 +1,82 @@
+from app.config import settings
+from app.dependencies.auth import UserContextDep
+from app.dependencies.entitysdk import get_client
+from app.logger import L
+
+from typing import Annotated
+
+from pydantic import BaseModel, Field
+from fastapi import Depends
+
+from pathlib import Path
+import io
+import tempfile
+
+import entitysdk.client
+import entitysdk.common
+
+from fastapi import APIRouter
+
+from entitysdk.models.morphology import ReconstructionMorphology
+from neurom import load_morphology
+import neurom
+
+from obi_one.scientific.morphology_metrics.morphology_metrics import MorphologyMetricsOutput
+
+def activate_declared_router(router: APIRouter) -> APIRouter:
+    
+    @router.get("/neuron_morphology_metrics/{reconstruction_morphology_id}", summary="Neuron morphology metrics", description="This calculates neuron morphology metrics for a given reconstruciton morphology.")
```

Comment:
can you put hyphen rather than underscore ? neuron_morphology_metrics -> neuron-morphology-metrics

Reply:
I mean for the end point, not the function